lib/rmt/cli/mirror.rb

```diff
@@ -1,6 +1,11 @@
 class RMT::CLI::Mirror < RMT::CLI::Base
   class_option :do_not_raise_unpublished, desc: _('Do not fail the command if product is in alpha or beta stage'), type: :boolean, required: false
 
+  class_option :ignore_gpg_checks, desc: _(
+    'Do not fail the command if repositories/products gpg signature ' \
+    'verification failed'
```

Comment:
What's the use case in which we don't want to fail the mirroring of a repo if the signature verification failed? 
By going over that exception we are breaking the security chain of our software delivery and could be mirroring tampered repository data, or do I get this wrong?

Reply:
@jesusbv ^^ that's the comment I was referring to. Even if that signature verification failure heals itself (for unknown reason) after some time, until then we would provide a repository to customers which failed the verification. 
In the light of recent software supply chain attacks, I wouldn't recommend you to take this risk.
It would be very interesting to know why this failure happens at all, it only appeared to happen since a few weeks right?

Reply:
@digitaltom I agree, it would be better to fix the root cause but that is happening in SCC, AFAIK, the repositories with wrong (as in it fails to verify) GPG signature
come from SCC so, not sure what is going on with those in SCC before being mirrored.

It's been happening for a while, maybe a month or so, so recent but not that recent.

Reply:
SCC is not managing the CDN content at updates.suse.com, the content gets directly mirrored there from the build service. 
So it would be great to have a snapshot of the repo content from the moment of the failure to reproduce, and also very detailed logs from the RMT that had the failure. 
We did not get any report about this from customers, could this be anyhow related to the way the public cloud RMTs access the CDN?